services/ux-backend/handlers/rotatekeys/handler.go

```diff
@@ -0,0 +1,71 @@
+package rotatekeys
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	onboardingValidationPublicKeySecretName = "onboarding-ticket-key"
+	podNamespaceEnvVar                      = "POD_NAMESPACE"
+)
+
+var namespace string = func() string {
+	if ns := os.Getenv(podNamespaceEnvVar); ns != "" {
+		return ns
+	}
+	panic(fmt.Sprintf("Value for env var %q is not set", podNamespaceEnvVar))
+}()
```

Comment:
Can we have a func in `handlers/utils.go` (instead of the `constants.go` file, to expose the operator namespace?
We can calc it once and store it in a local var within the handler module/

Reply:
done.